pom.xml

```diff
@@ -8,7 +8,7 @@
     <packaging>jar</packaging>
 
     <name>hyperwallet-java-sdk</name>
-    <description>An SDK implementation in Java for the v3 REST APIs.</description>
+    <description>An SDK implementation in Java for the v4 REST APIs.</description>
```

Comment:
noticed that version is still  `<version>1.4.3-SNAPSHOT</version>` I thought we reserved `1.x.x` as V3 and `2.x.x` as V4 ? or any equivalent?

Reply:
That's right, I updated Hyperwallet.java, README.md and CHANGELOG.md with 2.0.0 as suggested by Willian. 
He said it was ok to let the version stay the same in pom.xml for now. 
I believe the version will be updated by the integrator.

Reply:
I pushed the changes once again, looks like they didnt make it last time.